.github/workflows/equinix_metal_flow.yml

```diff
@@ -100,11 +100,19 @@ jobs:
           git add docs/*
           git commit -m "Add validation report for ${DATE_STR}-${TIME_STR}" -s
           git push
+          echo "date=${DATE_STR}" >> $GITHUB_OUTPUT
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           TOTAL_RUNTIME_SECONDS: 1200
           VALIDATOTR_CURVE_TYPE: "default"
 
+      - name: upload to artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: equinix_metal
+          path: docs/validation/${{ steps.run_workload.run_playbook.date }}
```

Comment:
Why not just upload snapshot and logs instead of complete directory?

Reply:
currently we org the result by folder with name convention as `docs/validation/{date}` and there seems lots of data in the folder for example
```
yuanyi@yuanyideMacBook-Pro validator-v0.7.12-45-gf434c667-11-37-50 % ls -al
total 64
drwxr-xr-x   9 yuanyi  staff   288 Nov 20 14:03 .
drwxr-xr-x   4 yuanyi  staff   128 Nov 20 14:03 ..
-rw-r--r--@  1 yuanyi  staff  6148 Nov 20 14:07 .DS_Store
drwxr-xr-x  20 yuanyi  staff   640 Nov 20 14:01 artifacts
drwxr-xr-x  11 yuanyi  staff   352 Nov 20 14:01 images
drwxr-xr-x   4 yuanyi  staff   128 Nov 20 14:04 prometheus-snapshot
-rw-r--r--   1 yuanyi  staff    20 Nov 20 14:01 rapl-domain-availability.txt
-rw-r--r--   1 yuanyi  staff  8230 Nov 20 14:01 report-v0.7.12-45-gf434c667.md
-rw-r--r--   1 yuanyi  staff  4852 Nov 20 14:01 v0.7.12-45-gf434c667.json
```
at meanwhile, we also need to update the python code as https://github.com/sustainable-computing-io/kepler-metal-ci/blob/main/util/validation_regression_check.py#L11
so, as 1st step, in this PR, just upload the folder to artifact.